scripts/xtensa-build-zephyr.py

```diff
@@ -361,12 +375,23 @@ def build_platforms():
 	execute_command(["west", "status", "hal_xtensa", "sof"], cwd=west_top,
 			stdout=subprocess.DEVNULL)
 
+	if validate_key_name(args.key_name):
+		signing_key_name = args.key_name
+	else:
+		raise RuntimeError("invalid signing key name: " + args.key_name)
```

Comment:
@marc-hb  Since we're going to use --key-type, and I think each vendor will have limited key types, so we can define a key type list as @aiChaoSONG suggested and let argparse do the work. So we no longer need  `validate_key_name` and `import re`.